.babelrc

```diff
@@ -1,16 +1,13 @@
 {
   "presets": [
-    ["env", {
+    ["@babel/preset-env", {
       "targets": {
-        "electron": "1.8"
-      },
-      "useBuiltIns": true
-    }], "react"
+        "electron": "3.0"
+      }
+    }], "@babel/preset-react"
   ],
   "plugins": [
-    "transform-decorators-legacy",
-    "transform-class-properties",
-    "transform-object-rest-spread",
-    "transform-runtime"
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
```

Comment:
When using `legacy` decorators, `loose` is important to use on `class-properties` extension according to official docs: https://babeljs.io/docs/en/babel-plugin-proposal-decorators